ines/43.js

```diff
@@ -100,7 +99,37 @@ function wordRank(string){
         }
     return result;
 }
-console.log("5==>",wordRank("Abcd"));
+console.log("wordValue==>",wordValue("fox"));
+
+function wordRank(sentence){
+    let sentenceArray = sentence.split(" ")
+    let arrayOfWordValues = [];
+    for(word of sentenceArray){
+        arrayOfWordValues.push(wordValue(word));
+    }
+    arrayOfWordValues.reduce( (acc,curr) => {
+        if(curr>acc){
+            acc = curr;
+        }else{
+            acc+=0;
+        }
+        return acc;
+    },0)
+    let indexOfAcc = arrayOfWordValues.indexOf(71)
+    return sentenceArray[indexOfAcc];
+}
+console.log(wordRank("The quick brown fox."));
+
+/* function reduceBiggest (arrayOfNums){
```

Comment:
Task 5 ==> wordRank("The quick brown fox.") ➞ "brown" is correct but if I try to check wordRank("Nancy is very pretty.") ➞ "pretty" the output is  very instead of pretty.
- And also if I check the last 2 below conditions the output will be undefined
wordRank("Check back tomorrow, man!") ➞ "tomorrow"
wordRank("Today is Wednesday.") ➞ "Wednesday"

I give you the solution because you already tried. Please take a look and add it to your code.
//5
const alphabet = ["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","w","v","x","y","z"];
const alphabetBig = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","W","V","X","Y","Z"]; 
function wordRank(string){
    stringArray = string.split("");
    let result = 0; 
        for(letter of stringArray){
            alphabet.find(function (element, index){
                if(element === letter){
                    result += index +1;
                }  
            }); 
            alphabetBig.find(function (element, index){
                if(element === letter){
                    result += index +1;
                }  
            }); 
        }
    return result;
}
console.log("5==>",wordRank("Abcd "));
console.log("5==>",wordRank("The quick brown fox.")) //➞ "brown"
console.log("5==>",wordRank("Nancy is very pretty.")) //➞ "pretty"
console.log("5==>",wordRank("Check back tomorrow, man!")) //➞ "tomorrow"
console.log("5==>",wordRank("Today is Wednesday.")) //➞ "Wednesday"